### PR TITLE
Avg & Max response summary

### PR DESCRIPTION
**Pull request by yjqg6666**
A new PR which is rebased on the current master. The old PR is #7115.

### CODE REVIEW

**Comment by binDongKim** [comment]
Is there anything you have changed in UI side since the last review?

**Comment by yjqg6666** [comment]
> Is there anything you have changed in UI side since the last review?

Nothing changed, just git-rebased on the current master.

**Comment by emeroad** [comment]
image

@yjqg6666
Thanks for your contribution
The avg/max values ​​are not displayed in the servermap
Please review this PR

**Comment by yjqg6666** [comment]
The PRed version will not show avg/max info on the edges of the servermap, only on the dedicated chart.

**Comment by yjqg6666** [comment]
Do you mean the ServerMap illustrated in #7212?

**Comment by yjqg6666** [comment]
If so and this feature is needed, i will append the related commits ASAP.

**Comment by emeroad** [comment]
Okay. This feature will be merged in 2.3.0

**Comment by yjqg6666** [comment]
@binDongKim @emeroad Just added the feature in the commit 7a2897f2f59c27c6147e180862453c23fd10383a. Please take a look.

File: web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts [hunk 1]
```diff
@@ -153,12 +153,40 @@ export class ServerMapDiagramWithCytoscapejs extends ServerMapDiagram {
...
+            hint += "avg:" + histogram_avg + "ms ";
+        }
+        if (histogram_max > 0) {
+            hint += "max:" + histogram_max + "ms ";
+        }
+        hint = hint.substring(0, hint.length - 1);
+        return hint + ") ";
+    }
+
+
```
Comment (binDongKim) [inline web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts]: - Please use single quotes for string for the consistency.
- Give a space before `0` in line 161.

**Review by binDongKim — CHANGES_REQUESTED** [review]
I've left a simple comment. Thanks for your contribution.

**Review by binDongKim — APPROVED** [review]

**Pull request merged**